rails_event_store/lib/rails_event_store/active_job_dispatcher.rb

```diff
@@ -49,9 +51,10 @@ def initialize(proxy_strategy: AsyncProxyStrategy::Inline.new)
       @async_proxy_strategy = proxy_strategy
     end
 
-    def call(subscriber, event)
+    def call(subscriber, payload)
       if async_handler?(subscriber)
-        @async_proxy_strategy.call(subscriber, event)
+        _, serialized_event = *payload
```

Comment:
no idea what's going on here? are we passing an array as `payload` here?

Reply:
if so, why not go with

```
call(subscriber, event, serialized_event)
```

?

Reply:
Sync handlers need event, Async handlers need serialized_event,I though it might be better idea to just keep them together as one argument and  unpack them depending on the dispatcher implementation instead of passing them always as separate arguments and then experiencing mutant brag about one or another being not used. 

But I'm not really opinionated here, we can go with explicit arguments and figure out how to deal with mutant either by providing right subject spec (optimal) or ignoring Dispatcher#call at all (suboptimal)